manuscript.md

```diff
@@ -12,6 +16,30 @@
 
 ## What is currently limiting our ability to predict network structure?
 
+* As this will probably play a part in up setting the tone/concept for the ms 
+I guess it makes sense to make sure we address the need for ‘tools’ (as 
+granted by the all seeing light of ML) That being said might be nice to bring 
+in a general point for each section - if at all possible...
+
+The scarcity of data on species interactions is certainly one of the biggest 
+barriers in predicting network structure. Machine learning methods, especially deep 
+learning methods, need a lot of data. Compounding the scarcity of data the 
+data that are available are biased toward food webs and select mutualistic 
+networks, and could prove to be a limiting factor when trying to understand or
+predict networks of *other* interaction types. A better understanding of how 
+different types of networks differ from each another would serve as a starting 
+point, although this in itself could be a potential hurdle 
+[@Michalska-Smith2019TelEco].
```

Comment:
this is a good point but under-developed - why is it important, and how is it going to help? If it's a starting point, what is next?